README.md

```diff
@@ -33,7 +33,8 @@ There are three way to install.
 
 **After installation, you should open the `Swimat.app` once to make the functionality works.**
 
-In the menu open **[Editor] -> [Swimat] -> [Format]** or map it to a hot-key you like in Xcode's Key Binding preferences.
+In the Xcode menu click **[Editor] -> [Swimat] -> [Format]** then the current active file will reformat.
+You can also create a hot-key in **[Xcode] -> [Preferences..] -> [Key Bindings]**, if you don't have any prefernce you can set as [⌘ + ⇧ + L].
```

Comment:
Does GitHub markdown render `<kbd></kbd>`?

Reply:
I think so and good point.

Reply:
<img width="1117" alt="Screen Shot 2019-09-06 at 11 52 57 PM" src="https://user-images.githubusercontent.com/3413874/64441948-7e8f3e00-d101-11e9-806d-248f7d8bfdb6.png">
Though it's not as beauty as I expect.

Reply:
Yeah, it's not the nicest looking CSS :(